docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -44,6 +67,365 @@ To utilize this store you will import the `CART_STORE_KEY` in any module referen
 const { CART_STORE_KEY } = window.wc.wcBlocksData;
 ```
 
+## Actions
+
+### setCartData
+
+This action is used to set the cart data in the store.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   `cartData: Object`: The new cart data.
```

Comment:
For consistency-reasons, I suggest using the same formatting as we use in our other docs, e.g.
https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/data-store/collections.md?plain=1#L33-L40

What are your thoughts on documenting the inner structure of the `cartData` object, similar to what we're doing in https://github.com/woocommerce/woocommerce-blocks/blob/6960ae656112b5a5249673db2aaa80e13c4b94c7/docs/third-party-developers/extensibility/data-store/cart.md?plain=1#L188-L195